pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/MultiStageEngineIntegrationTest.java

```diff
@@ -485,6 +486,25 @@ public void testLiteralOnlyFunc()
     assertEquals(results.get(10).asText(), "hello!");
   }
 
+  @Test
+  public void testMultiValueColumnGroupBy()
+      throws Exception {
+    String pinotQuery = "SELECT count(*), arrayToMV(RandomAirports) FROM mytable "
```

Comment:
BaseIntegrationTestSets also have (1) hard-coded MV tests queries; and (2) withMultiValue flag in generated queries
should we turn them on?

Reply:
v1/v2 behavior are different, we can turn on them.

Reply:
done